SocialCareCaseViewerApi.Tests/V1/Factories/EntityFactoryTests.cs

```diff
@@ -541,33 +541,6 @@ public void CaseSubmissionToCareCaseDataReturnsDateOfEventAsCreatedAtIfDateOfEve
 
             response.DateOfEvent.Should().Be("2021-07-18T14:40:30.0000000");
         }
-
-        [Test]
-        public void CaseSubmissionToCareCaseDataReturnsTitleAndFormIdForFormNameWhenTitleProvided()
```

Comment:
Why are removing these tests?

Reply:
We no longer provide the title in the FormName property, we return an optional title property instead.